solidity/.soliumrc.json

```diff
@@ -1,19 +1,16 @@
 {
-    "extends": "solium:recommended",
-    "plugins": [
-      "security"
+  "extends": "keep",
+  "plugins": [
+    "security"
+  ],
+  "rules": {
+    "security/no-block-members": [
+      "off",
+      [
+        "timestamp"
+      ]
     ],
-    "rules": {
-      "quotes": [
-        "error",
-        "double"
-      ],
-      "indentation": [
-        "error",
-        4
-      ],
-      "security/no-block-members": ["off", ["timestamp"]],
-      "security/no-call-value": "off",
-      "security/no-inline-assembly": "off"
-    }
-}
\ No newline at end of file
+    "security/no-call-value": "off",
+    "security/no-inline-assembly": "off"
+  }
```

Comment:
We should consider unifying these rules at the `keep` level, as a side note. Haven't peeked into this yet, so not a blocking note or anything.